static/js/lib/api.js

```diff
@@ -1,137 +1,18 @@
 // @flow
-/* global SETTINGS:false, fetch: false */
-// For mocking purposes we need to use 'fetch' defined as a global instead of importing as a local.
-import 'isomorphic-fetch';
-import R from 'ramda';
+import { fetchJSONWithCSRF } from 'redux-hammock/django_csrf_fetch';
 
-import { S, parseJSON, filterE } from './sanctuary';
-
-export function getCookie(name: string): string|null {
```

Comment:
Are these functions being cut because they're unlikely to ever be useful for odl-video?